CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [FEATURE] Alertmanager: Added `-alertmanager.max-silences-count` and `-alertmanager.max-silence-size-bytes` to set limits on per tenant silences. Disabled by default. #6898
```

Comment:
Did you mean to add this here?

Reply:
I don't see this change as part of this PR 🤔

Reply:
It's not. It was just in the wrong place in the changelog and it got confusing. We fixed it in https://github.com/grafana/mimir/pull/8266